RecoBTag/CTagging/src/CharmTagger.cc

```diff
@@ -7,10 +7,12 @@
 #include "CondFormats/DataRecord/interface/GBRWrapperRcd.h"
 
 #include <iostream>
-#include <vector>
+#include <memory>
+
 #include <algorithm>
-#include <map>
 #include <iostream>
```

Comment:
duplicate

Reply:
I dont't see which one is the duplicate here, memory is included once

Reply:
ah yes its iostream, ill fix it